test/kb_quast_server_test.py

```diff
@@ -292,15 +289,15 @@ def test_fail_bad_quast_input_garbage(self):
         self.fail_quast(
             {'files': [{'path': 'data/greengenes_UnAligSeq24606.fa'},
                        {'path': 'data/greengenes_UnAligSeq24606_garbage.fa'}]},
-            'QUAST skipped some files - 2 expected, 1 processed.')
+            "'utf-8' codec can't decode byte 0x8b in position 1: invalid start byte")
 
     def test_fail_bad_ws_ref(self):
         self.start_test()
         self.fail_quast(
             {'assemblies': [str(self.ws_info[0]) + '/999999999999/999999']},
             'No object with id 999999999999 exists in workspace {} (name {})'.format(
                 self.ws_info[0], self.ws_info[1]),
-            exception=WorkspaceError)
+            exception=WorkspaceError, contains=True)
```

Comment:
What changed such that contains is necessary?

Reply:
These calls return a traceback with the error in Python3

Reply:
So the user is going to get a huge traceback in the narrative instead of a small error string? That doesn't seem right.